docs/validation/daily-report/2023-11-27/index.md

```diff
@@ -0,0 +1,92 @@
+# 2023-11-27: Transect NSW Site 1, Sentinel 2b overpass
+
+A Daily Validation Summary Report of the surface reflectance data collected on the date of 2023-11-27 by Geoscience&nbsp;Australia. 
+The full collection of data is contained in the [National Spectral Database](https://www.dea.ga.gov.au/products/national-spectral-database).
+
+:::{contents} In this report
+:local:
+:backlinks: none
+:::
+
+## Variables
+
+These variables and environmental factors were present on the day the data were collected.
+
+:::{csv-table}
+:class: validation-report-variables-table
+
+"Instrument(s) used","ASD FR4 (18179/3)"
+"Time of field site measurements (UTC)","2023-11-26 22:37:46 to 2023-11-26 23:37:06"
+"Time of overpass (UTC)", 2023-11-27 00:45:46
+"GPS quality","Good"
+"Reference position","142.1036962E, 31.8135401S (WGS84)"
+"Matchup quality","Good"
+:::
+
+## Results
+
+The surface reflectance statistics collected on this day are grouped into bands. 
+This allows the data to be used to validate Geoscience Australia's other datasets which use the same standardised bands.
```

Comment:
```suggestion
This allows the data to be used to validate Geoscience Australia's other datasets that use the same standardised bands.
```

I think that not all GA datasets use the same standardised bands (is that correct?), therefore, the word "that" is more grammatically correct here. Sorry, I didnt notice that before.